051-MicrosoftPurview/Student/Challenge6.md

```diff
@@ -0,0 +1,25 @@
+# Challenge 6: Data Lineage
+
+[< Previous Challenge](./Challenge5.md) - [Home](../README.md) - [Next Challenge >](./Challenge7.md)
+
+## Introduction
+Data engineers in Fabrikam are working on a new ETL process which will transform data gathered from multiple sources and will be consumed later by Power BI report. Engineers want to represent this process visually, so it is available for further analysis like troubleshooting, tracing root cause in data pipelines or impact analysis.
+
+## Description
+With multiple data sources in your environment, it may become challenging to understand lifecycle of the whole data transformation process. As these sources may be dependent on each other, changing one data set can impact the whole process. So far, you have scanned your data sources and you can view individual metadata about these datasets. Time to see the dependency and flow of data in your environment. It can be extremely helpful for "what if" analysis, tracing root cause or just to understand the lifecycle of the data. 
```

Comment:
Add "the" between "understand" and "lifecycle"

Reply:
Done